pages/components/Profile/ProjectsCard.tsx

```diff
@@ -24,7 +24,7 @@ const ProjectsCard: React.FC<Props> = ({ projectImage, projectName, projectDesc,
                     backgroundRepeat: "no-repeat",
                     backgroundSize: "cover"
                 }} />
-            <div className="p-5 h-[17rem] flex flex-col">
+            <div className="p-5 h-[21rem] flex flex-col">
```

Comment:
I added 4rem to card height; I think if you want a more or less consistent look, this makes sense since 3 long tags take up too much space if the height is 17rem. but that is 100% up for debate, of course.

Reply:
It's good I guess, no problem here as well.